app/components/note-list/note-list-item.jsx

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
```

Comment:
Please move this to a separate folder (components/note-item). We are trying to isolate each individual component to ensure ease of understanding for beginners.

Reply:
done, changed the path so it would work. also now eslint passes but im unable to send another pull request. maybe you should refuse the first one to ba able to merge 
thanks!